requirements/edx/github.in

```diff
@@ -74,3 +74,7 @@ git+https://github.com/edx/django-ratelimit-backend.git@v2.0.1a5#egg=django-rate
 
 git+https://github.com/open-craft/xblock-poll@1efd04bd6e16252a20e39a7516f9b69a000ace24#egg=xblock-poll==1.10.0
 git+https://github.com/edx-solutions/xblock-drag-and-drop-v2@v2.2.10#egg=xblock-drag-and-drop-v2==2.2.10
+
+
+# Test new version of edx-search
+git+https://github.com/raccoongang/edx-search.git@golub-sergey/BD-19/transition-from-ES1.5-to-ES7
```

Comment:
```suggestion
git+https://github.com/raccoongang/edx-search.git@80980366434e701bc8c0524600bd595e6704b51f
```

We can lock this to a specific commit (latest in https://github.com/edx/edx-search/pull/104) for the stage test. Once that looks good, we can cut an edx-search release and switch to using that in edx-platform.

Reply:
@timmc-edx Done